### ISSUE TITLE
menu bug

### ISSUE THREAD

**Issue by nycnoise** [gregsadetsky/nycnoise#365]
minor, & probably won't come up for most users (i'm just toggling between things to copy/paste stuff today):

with the menu expanded on the homepage (on laptop), toggling between multiple pages –– say, clicking on "tenants" & then "unions" & then "misc" –– sometimes results in a "Not Found" because the site's adding subfolders...?

![Screenshot 2024-12-19 at 1 51 07 PM](https://github.com/user-attachments/assets/2ee70578-604a-446a-9fcb-39a7962fd51f)

(i have so much to catch up on otherwise here eep!!!)

**Comment by gregsadetsky** [comment]
good catch!

**Comment by gregsadetsky** [comment]
the easiest fix for now is to change the nonprofits and the unions url's (in the database) to `/nonprofits` and `/unions` respectively

once/if all of the database urls start with `/`, we could just add the / automatically in the template and remove them from all of the database items (all of the urls could just be "nonprofits", "unions", etc.)

otherwise, you always have to remember to add that first slash.

TLDR - you can fix it with 2 database edits right now but yeah, there's a cleaner solution longer term

**Comment by nycnoise** [comment]
it looks like *none* of the static pages have ```/``` in the url path –– but that makes me think you're probably talking about something else...?

(read as, "is this something _i_ can fix?")

**Comment by gregsadetsky** [comment]
ahh yeah sorry you're right

I just pushed a fix that should fix this

**Comment by gregsadetsky** [comment]
ah fuck I'm dumb that won't work yeah... so yeah no, I'll revert my change.

you *should* add a `/` at the beginning the menu items in the database i.e. in https://nyc-noise.com/admin/core/menuitem/

**Comment by gregsadetsky** [comment]
really the problem was that

https://nyc-noise.com/admin/core/menuitem/29/change/

i.e. the nonprofits menu item did not start with a `/` -- the url needs to be `/nonprofits`, which it is now

**Comment by gregsadetsky** [comment]
fixed, and going to also deploy a small change to add a bit of help text that will make it a bit clearer hopefully

<img width="980" alt="image" src="https://github.com/user-attachments/assets/aaadba2c-a2f6-4ca0-81c6-7a542d36b44e" />
